css/defold.scss

```diff
@@ -244,6 +244,16 @@
 select {
 	overflow:auto;
 }
+.download-select {
+	background: url("data:image/svg+xml;utf8,<svg xmlns='http://www.w3.org/2000/svg' width='100' height='100' fill='%23FFFFFF'><polygon points='0,0 100,0 50,50'/></svg>") no-repeat;
+	background-size: 10px;
+	background-position: calc(100% - 10px) calc(50% + 3px);
+    -moz-appearance: none; 
+    -webkit-appearance: none;
+    appearance: none;
+	overflow: hidden;
+	width: 200px;
```

Comment:
I had to hardcode the width because Firefox couldn't fit the whole text for some reason.